src/components/base-address/BaseAddress.stories.ts

```diff
@@ -18,26 +18,30 @@ const Template = (args: AddressIF) => ({
 
 export const Default = Template.bind({})
 Default['args'] = {
-  street: '',
-  streetAdditional: '',
-  city: '',
-  region: '',
-  postalCode: '',
-  country: '',
-  deliveryInstructions: '',
+  value: {
+    streetAddress: '',
+    streetAddressAdditional: '',
+    addressCity: '',
+    addressRegion: '',
+    addressCountry: '',
+    postalCode: '',
+    deliveryInstructions: ''
+  },
   schema: DefaultSchema,
   editing: true
 }
 
 export const WithData = Template.bind({})
 WithData['args'] = {
-  street: '1234 Sesame Street',
-  streetAdditional: '4th Floor',
-  city: 'Victoria',
-  region: 'BC',
-  postalCode: 'V8N 1A1',
-  country: 'Canada',
-  deliveryInstructions: 'Leave at front door',
+  value: {
```

Comment:
The story for BaseAddress was wrong since the props were incorrect. The new one from assets uses "value". 

Before:
![before with data](https://github.com/bcgov/bcrs-shared-components/assets/122301442/a738582d-eb00-4ef9-8b8c-c15b38911e0a)

Now:
![after with data](https://github.com/bcgov/bcrs-shared-components/assets/122301442/3ed23d31-491b-47cb-a294-2243b9368c70)